### DISCUSSION TITLE
怎么运行啊？

### DISCUSSION THREAD

**Discussion by Vincent-1125** [leiurayer/downkyi#330] [Q&A]
怎么运行啊
[README.md](https://github.com/leiurayer/downkyi/files/8458195/README.md)
？

**Comment by leiurayer** [comment]
下载：
https://github.com/leiurayer/downkyi/releases/latest
解压zip，
双击运行DownKyi.exe

**Reply by HighVorz** [reply]
怎麽在VS上部署項目？
